zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -1012,12 +1012,12 @@ public void setNoteRevision(String noteId,
 
   }
 
-  public void getNotebyRevision(String noteId,
+  public Note getNotebyRevision(String noteId,
```

Comment:
Could you please put the following comment to this method.
`notebook.getNoteByRevision(...) does not use the NoteCache, so we can return a Note object here.`

Reply:
> Could you please put the following comment to this method. `notebook.getNoteByRevision(...) does not use the NoteCache, so we can return a Note object here.`

Sure! Thanks a lot for your review!